app/scripts/services/logLinks.js

```diff
@@ -73,7 +73,10 @@ angular.module('openshiftConsole')
       ].join(''));
 
 
-      var archiveUri = function(opts, prefix) {
+      var archiveUri = function(opts, prefix, canViewOperationsLogs) {
+        if((canViewOperationsLogs) && (!prefix || prefix.startsWith('project.'))) {
```

Comment:
style nit: i think the extra parens around canViewOperationsLogs actually make it harder to read